src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1893,7 +1894,7 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
                     expect(error.message).toBe('Assertion failure: ' + expectedError);
                     failed = true;
                 }
-                expect(failed).toBeTrue();
+                expect(failed).toBeTrue(); // TODO FOR REVIEW: this one is the last remaining monster!
```

Comment:
à investiguer avant merge si t'arrives pas

Reply:
I'VE SLAINED THE BEAST

Reply:
YES CONGRATS ! C'était quoi le soucis ?

Reply:
les async et les pas async, n'ont OLOLELOL bizarrement pas le même behavior, comme tu vois dans le code j'ai séparé en deux